docker-compose.yml

```
@@ -0,0 +1,13 @@
+services:
+  montage:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "5001:5000"
```

any particular reason why we remap this here?

---

On Macbook, AirPlay functionality runs on port 5000. This always make problem to new developer. See https://forums.developer.apple.com/forums/thread/682332

---

Ohhh that's a good callout. Can we comment that in the file then?

```suggestion
      - "5001:5000"  # Airplay runs on port 5000 on mac. See https://forums.developer.apple.com/forums/thread/682332
```